index.html

```diff
@@ -179,12 +196,13 @@ <h2>Multi-Post Stories</h2>
                     <div>
                         <button class="project-button">See Project</button>
                     </div>
+                </section>
                 </div>
             </section>
 
             <section class="about">
                 <div class="about-container">
-
+                <section class="about-headings">    
                     <h2 class="about-heading">About
                         Myself
                     </h2>
```

Comment:
- [ ] Kindly follow the same `font-size` and text `color` as shown on Figma, as well as button interaction styles 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194669491-fad3dc9c-95a8-4a22-9c4b-90d4e79ccd84.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194669517-165bb25c-f678-436a-9ae3-95f0c258fa00.png)